src/analysis/inner_product_space/gram_schmidt_ortho.lean

```diff
@@ -104,6 +113,21 @@ theorem gram_schmidt_pairwise_orthogonal (f : ι → E) :
   pairwise (λ a b, ⟪gram_schmidt 𝕜 f a, gram_schmidt 𝕜 f b⟫ = 0) :=
 λ a b, gram_schmidt_orthogonal 𝕜 f
 
+lemma gram_schmidt_inv_triangular (v : ι → E) {i j : ι} (hij : i < j) :
```

Comment:
This naming isn't obvious to me; can you add a docstring that explains it? Is it referring to triangular in the matrix sense, as opposed to the inner product triangle rule sense?

Reply:
Can you please transfer this comment to the dependent PR #16477?